docs/source/pipeline.rst

```diff
@@ -0,0 +1,291 @@
+
+=============
+Documentation
+=============
+
+Ingestion
+~~~~~~~~~
+
+| For more information on how to customize this step: `DataLoader Parameters <https://docs.vertica.com/23.4.x/en/sql-reference/statements/create-statements/create-data-loader/#arguments>`__ and `Copy Parameters <https://docs.vertica.com/23.4.x/en/sql-reference/statements/copy/>`__.
+
+.. code:: yaml
+
+     ingest:
+         from: '~/data/bucket/*'
+         delimiter: ','
+         retry_limit: 'NONE'
+         retention_interval: "'15 days'"
+
+Transform
+~~~~~~~~~
+
+| For more information on how to customize this step: `Transform Functions <https://www.vertica.com/python/documentation/1.0.x/html/vdataframe.html#features-engineering>`__.
+
+| ``public.winequality``
+
+    .. ipython:: python
+        :suppress:
+
+        from verticapy.datasets import load_winequality
+        html_file = open("SPHINX_DIRECTORY/figures/datasets_loaders_load_winequality.html", "w")
+        html_file.write(load_winequality()["density", "pH", "color", "fixed_acidity"]._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: ../SPHINX_DIRECTORY/figures/datasets_loaders_load_winequality.html
+        
+ 
+
+**Example**
+
+.. code:: yaml
+
+   transform:
+       # 1. Existing Column
+       col1:
+           sql: fixed_acidity
+       # 2. Column + Transform
+       col2:
+           sql: color
+           transform_method:
+               name: str_count
+               params:
+                   pat: white
+       # 3. Create a new column with Method
+       # Note: Don't specify 'name' in params
+       col3:
+           transform_method:
+               name: cummax
+               params:
+                   column: fixed_acidity
+       # 4. Complex Sql
+       col4:
+           sql: fixed_acidity * density
+       # 5. Multiple Params
+       col5:
+           transform_method:
+               name: regexp
+               params:
+                   column: color
+                   pattern: "white"
+                   method: "replace"
+                   replacement: "NOT white"
+       # 6. Multi-Stage Transforms
+       col6:
+           sql: color
+           transform_method1:
+               name: str_count
+               params:
+                   pat: white
+           transform_method2:
+               name: add
+               params:
+                   x: 0.5
+       # 7. Using Previously Created Columns
+       col7:
+           sql: col2
+           transform_method:
+               name: add
+               params:
+                   x: 0.5
+
+Train
+~~~~~
+
+| For more information on how to customize this step: `Models <https://www.vertica.com/python/documentation/1.0.x/html/machine_learning_vertica.html#api-machine-learning-vertica>`__.
+| If you want to train a model, the default setting is to use all the previously created **cols** as predictors.
+| To subtract the specified columns from the default columns use **exclude**.
```

Comment:
for parameters, we have previously used ``param`` instead of making it bold. The code-type output will be more consistent if you can switch. 

e.g. ``exclude`` , ``include`` etc